Dart/src/com/jetbrains/lang/dart/ide/runner/server/DartCommandLineRunningState.java

```diff
@@ -126,7 +127,27 @@ public void onTextAvailable(final ProcessEvent event, final Key outputType) {
       }
     });
 
+    // Check for and display any analysis errors when we launch a Dart app.
+    final Project project = getEnvironment().getProject();
+    try {
+      final DartRunConfiguration dartRunConfiguration = (DartRunConfiguration)getEnvironment().getRunProfile();
+      final VirtualFile launchFile = dartRunConfiguration.getRunnerParameters().getDartFileOrDirectory();
+
+      if (getEnvironment().getRunnerAndConfigurationSettings() == null) {
+        DartExecutionHelper.displayIssues(project, launchFile);
+      }
+      else {
+        String launchTitle = "Analysis issues with " + getEnvironment().getRunnerAndConfigurationSettings().getName();
```

Comment:
`getEnvironment().getRunnerAndConfigurationSettings().getName()` seems to be equivalent to `dartRunConfiguration.getName()` which has NPE-safe signature. Rewriting it makes `DartExecutionHelper.displayIssues(Project, VirtualFile)` unused. Are you ok with me deleting it?